docs/en/getters.md

```diff
@@ -61,6 +61,27 @@ computed: {
 }
 ```
 
+You can also return functions in getters:
```

Comment:
It would be nice if there is a statement that describes this pattern is for passing some arguments to getters. :slightly_smiling_face: